study.md

```diff
@@ -151,5 +210,7 @@ Write some Ruby code that uses the Array `map` method and code blocks to
 duplicate this functionality.
 
 ```ruby
-# your answer here
+numbers = [1,2,3,4,5,6,7,8,9,10];
+
+squares = Array.new(numbers.length){ |i| i * 2) }
```

Comment:
The extraneous closing parenthesis here will cause an error.  Even without it, this just creates a 10-element array filled with `nil` values while the block iterates through the indices and returns the index doubled.